web/src/main/webapp/xslt/common/base-variables.xsl

```diff
@@ -76,7 +76,7 @@
     else if ($service = 'catalog.viewer') then 'gn_viewer'
     else if ($service = 'catalog.search') then 'gn_search'
     else if ($service = 'md.viewer') then 'gn_formatter_viewer'
-    else 'gn'"/>
+    else 'gn_search'"/>
```

Comment:
In some project, we have css classes and behavior matched on the `.gn` selector.
This will break this.

Could you keep the `gn` class at least ?